diff -uprN linux-5.4.108.old/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts linux-5.4.108/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts
--- linux-5.4.108.old/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts	2021-03-24 13:26:46.000000000 +0300
+++ linux-5.4.108/arch/arm/boot/dts/sun8i-h2-plus-orangepi-zero.dts	2021-03-31 02:59:09.628064358 +0300
@@ -124,6 +124,14 @@
 	status = "okay";
 };
 
+&ehci2 {
+	status = "okay";
+};
+
+&ehci3 {
+	status = "okay";
+};
+
 &emac {
 	phy-handle = <&int_mii_phy>;
 	phy-mode = "mii";
@@ -162,6 +170,14 @@
 	status = "okay";
 };
 
+&ohci2 {
+	status = "okay";
+};
+
+&ohci3 {
+	status = "okay";
+};
+
 &spi0 {
 	/* Disable SPI NOR by default: it optional on Orange Pi Zero boards */
 	status = "disabled";
