diff -uprN procd-2019-05-30-ade00ca5.old/initd/zram.c procd-2019-05-30-ade00ca5/initd/zram.c
--- procd-2019-05-30-ade00ca5.old/initd/zram.c	2019-05-30 08:53:26.000000000 +0300
+++ procd-2019-05-30-ade00ca5/initd/zram.c	2019-07-19 15:32:47.479441678 +0300
@@ -83,7 +83,8 @@ early_insmod(char *module)
 int
 mount_zram_on_tmp(void)
 {
-	char *mkfs[] = { "/usr/sbin/mkfs.ext4", "-b", "4096", "-F", "-L", "TEMP", "-m", "0", "/dev/zram0", NULL };
+	LOG("Preparing customized zram on /tmp mount\n");
+	char *mkfs[] = { "/usr/sbin/mkfs.ext4", "-q", "-b", "4096", "-F", "-L", "TEMP", "-m", "0", "-I", "128", "-O", "^has_journal,^dir_index,^large_file,^huge_file,^resize_inode,sparse_super2,^ext_attr,^dir_nlink,^extra_isize,uninit_bg", "-E", "discard,packed_meta_blocks=1,lazy_itable_init=1,num_backup_sb=0", "/dev/zram0", NULL };
 	FILE *fp;
 	long zramsize;
 	pid_t pid;
@@ -118,7 +119,7 @@ mount_zram_on_tmp(void)
 	}
 
 	if (!is_container()) {
-		ret = mount("/dev/zram0", "/tmp", "ext4", MS_NOSUID | MS_NODEV | MS_NOATIME, "errors=continue,noquota");
+		ret = mount("/dev/zram0", "/tmp", "ext4", MS_NOSUID | MS_NODEV | MS_NOATIME, "errors=continue,nobarrier,discard,delalloc,noauto_da_alloc,dioread_nolock");
 		if (ret < 0) {
 			ERROR("Can't mount /dev/zram0 on /tmp: %m\n");
 			return errno;
